nginx/nginx.conf

```diff
@@ -1,4 +1,4 @@
-user                    <%= nginxConf.NGINX_USER %> <%= nginxConf.NGINX_GROUP %>;
+user                    clozan01 staff;
```

Comment:
You shouldn't commit your own version of nginx as it depends on your own development environment